cityofaustin/atd-moped
# [Enhancement] Show component ID, council district(s), and length in component form

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/14157

## Testing
**URL to test:** 
https://deploy-preview-1154--atd-moped-main.netlify.app/moped/projects/1939?tab=map

**Steps to test:**

Edit an existing component's attributes. The modal should have some properties visible:

>We have some not-editable component properties that users need to be able to see in UI. They are component ID, council district, and length (in that order).

>The length field should use a thousands separator and be hidden if null.

The properties should not be visible when creating a new component


---
#### Ship list
